resources/views/events/index.blade.php

```diff
@@ -70,6 +70,12 @@
                         <td> {{ $event->alcohol ? 'Ja' : 'Nej' }}</td>
                     </tr>
                 @endif
+                @if ($event->entity->lol_question)
+                    <tr>
+                        <td>Behövas speciell LoL-utrustning:</td>
+                        <td> {{ $event->lol ? 'Ja' : 'Nej' }}</td>
+                    </tr>
+                @endif
```

Comment:
This is currently under `Information som inte visas för alla` since it was copied from Alcohol, but there might be marginal interest in making it publicly visible?